### ISSUE TITLE
Building the generator fails with the Xcode 10.2 toolchain

### ISSUE THREAD

**Issue by dmazzoni** [Brightify/Cuckoo#264]
It seems that building the generator always fails when using the Xcode 10.2 toolchain.

This is the relevant part of Cuckoo's log:

```
Fetching https://github.com/kylef/PathKit.git
Fetching https://github.com/Quick/Nimble.git
Fetching https://github.com/Carthage/Commandant.git
Fetching https://github.com/jpsim/SourceKitten.git
Fetching https://github.com/Quick/Quick.git
Fetching https://github.com/drmohundro/SWXMLHash.git
Fetching https://github.com/kylef/Stencil.git
Fetching https://github.com/jpsim/Yams.git
Fetching https://github.com/TadeasKriz/FileKit.git
Fetching https://github.com/kylef/Spectre.git
Fetching https://github.com/antitypical/Result.git
https://github.com/TadeasKriz/FileKit.git @ develop: error: manifest parse error(s):
/tmp/TemporaryFile.acKJOQ.swift:32:14: error: extra argument 'exclude' in call
    exclude: [
             ^
```

I suppose that the error is due to the fact that FileKit is written in Swift 3, which is not supported by the Xcode 10.2 compiler. I did not open a PR with the Swift 4 migration because FileKit's upstream repository (`nvzqz/FileKit`) has already been migrated to Swift 4.2.

**Comment by MatyasKriz** [comment]
Hi, thanks for checking, now it won't just stop working on us suddenly after using new Swift.
I investigated why we were using our own FileKit and only saw commits related to compilability in Xcode 9, which I'm sure is working in their master as of the moment. We discussed it and decided to switch back to the original FileKit.
Since you found this out, would you like to create a PR updating FileKit to point at the original repo again?

**Comment by dmazzoni** [comment]
At this moment it would only fix part of the issue. The original FileKit repo has already been migrated to Swift 4.2, but its `Package.swift` manifest file uses the v3 format, which seems to be incompatible with the Xcode 10.2 toolchain. In fact, the actual reason of the error I reported when opening this issue is the manifest file format, and not the project's Swift version.

Yesterday I forked FileKit and updated its `Package.swift` file so as to make it compatible with Xcode 10.0 to 10.2; before opening a PR on FileKit I'll also try to ensure compatibility with Xcode 9.4.
